physics/module_sf_mynn.F90

```diff
@@ -420,11 +404,19 @@ SUBROUTINE SFCLAY_mynn(                           &
 
       IF (itimestep==1 .AND. iter==1) THEN
          DO i=its,ite
-            !Everything here is used before calculated
-            UST_WAT(i)=MAX(0.04*SQRT(U1D(i)*U1D(i) + V1D(i)*V1D(i)),0.001_kind_phys)
-            UST_LND(i)=MAX(0.04*SQRT(U1D(i)*U1D(i) + V1D(i)*V1D(i)),0.001_kind_phys)
-            UST_ICE(i)=MAX(0.04*SQRT(U1D(i)*U1D(i) + V1D(i)*V1D(i)),0.001_kind_phys)
-            MOL(i)=0.0
+            IF (.not. flag_restart) THEN
+               !Everything here is used before calculated
```

Comment:
This comment catches my eye. The friction velocities for the various surface types are set in GFS_surface_composites_pre. For the first timestep in a cold start, all friction velocities are therefore set to zero coming in and thus the calculations are triggered. That is what is meant by "used before calculated", because friction velocities are then subsequently calculated in MYNN SFC, right? I'm just trying to understand if this comment is still valid.